src/nisarqa/processing/caltools/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .abscal import *
+from .noise_equivalent_backscatter import *
+from .pta import *
+from .pta_plotting import *
```

Comment:
FYI there's an inconsistency here w.r.t a previous PR -- unlike `src/nisarqa/input_product_readers/__init__.py`, there's no comment here noting the lack of `from ._utils import *`.

https://github.com/isce-framework/nisarqa/blob/40c1112a37fd1264d672fe8702392fada4dcd0a3/src/nisarqa/input_product_readers/__init__.py#L15-L19

IMO we should just remove that comment. It's not providing much value, and it seems tedious to copy-paste it every time we introduce an `_utils.py` module in a new subdirectory.